sunkit_image/utils/utils.py

```diff
@@ -19,6 +20,10 @@
     "points_in_poly",
     "reform2d",
     "remove_duplicate",
+    "_percentile_ranks_numpy_inplace",
+    "_percentile_ranks_numpy",
+    "_percentile_ranks_scipy",
```

Comment:
```suggestion
```

Since these are private, we typically don't want them to be in the "all" as that in theory defines the public API.

Reply:
I intend to pare this down to just the inplace and regular numpy version since the scipy version doesn't seem to be faster and adds the scipy dependency unnecessarily

Reply:
For now I'll just make them public